pkg/scalers/kafka_scaler.go

```diff
@@ -186,11 +187,14 @@ func (s *kafkaScaler) IsActive(ctx context.Context) (bool, error) {
 	}
 
 	for _, partition := range partitions {
-		lag := s.getLagForPartition(partition, offsets)
+		lag, err := s.getLagForPartition(partition, offsets)
+		if err != nil {
+			return true, nil
+		}
```

Comment:
only concern is, is any error really ok to saying isActive = true? Maybe we should check the exact error, in this case the invalid offset one?

Reply:
correct